src/docfx/lib/schema/JsonSchema.cs

```diff
@@ -129,5 +129,25 @@ internal class JsonSchema
         /// Properties that are used to realize precludes logic
         /// </summary>
         public string[][] Precludes { get; set; } = Array.Empty<string[]>();
+
+        /// <summary>
+        /// Properties that are used to validate date format
+        /// </summary>
+        public string DateFormat { get; set; }
+
+        /// <summary>
+        /// Properties that are used to set maximum time range
+        /// </summary>
+        public TimeSpan? RelativeMaxDate { get; set; }
+
+        /// <summary>
+        /// Properties that are used to set minimum time range
+        /// </summary>
+        public TimeSpan? RelativeMinDate { get; set; }
+
+        /// <summary>
+        /// Properties that are used to indicate the deprecated field
+        /// </summary>
+        public JValue ReplacedBy { get; set; }
```

Comment:
>JValue Repl [](start = 15, length = 11)

why JValue, not string?

Reply:
Because replaceBy can be set "", this means that field has been deprecated, but there is no other field to replace it. you can take a look at test for more details.

Reply:
A `string` can also be null or empty

Reply:
OK, I will fix this